include/license.php

```diff
@@ -84,10 +84,12 @@ public function __construct( $file, $item_name, $version, $author, $api_url = nu
 		$this->author  = $author;
 		$this->api_url = empty( $api_url ) ? $this->api_url : $api_url;
 
-		$licenses = get_option( 'polylang_licenses' );
-		$this->license_key = empty( $licenses[ $this->id ]['key'] ) ? '' : $licenses[ $this->id ]['key'];
-		if ( ! empty( $licenses[ $this->id ]['data'] ) ) {
-			$this->license_data = $licenses[ $this->id ]['data'];
+		$licenses          = (array) get_option( 'polylang_licenses', array() );
```

Comment:
What happen if `get_option( 'polylang_licenses' )` returns an object here?

Reply:
It becomes an array.

Reply:
Mmmmh... I thought casting an object into an array requires more work to get all properties properly. Thanks.